src/Entity/Programme.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\Collection;
+
+class Programme
+{
+    private int $id;
+
+    public string $name = '';
+
+    private \DateTime $startDate;
```

Comment:
You should probably have consistent naming on the DateTime properties. For example, in building you have startTime and here startDate. (I had the same issue myself :) )